probe-rs/src/probe/cmsisdap/tools.rs

```diff
@@ -169,12 +169,22 @@ pub fn open_v2_device(device_info: &DeviceInfo) -> Option<CmsisDapDevice> {
                 continue;
```

Comment:
Jlink CMSIS DAP enters here

Reply:
Oh hey, the one line I was sure it wouldn't hit :D

Unfortunately, I don't know what to expect from this device, so I'm not sure how to work around this.